src/ray/object_manager/plasma/common.h

```diff
@@ -45,23 +45,38 @@ enum class ObjectState : int {
   PLASMA_SEALED = 2,
 };
 
+// Represents a chunk of allocated memory.
```

Comment:
I think we can improve this message a little bit? What about "chunk of memory that corresponds to a single plasma object"?

Reply:
If you try hiding plasma object from here, you can update `ObjectTableEntry`'s comment that currently 1 allocation represents 1 plasma object entry.

Reply:
:) I think allocator shouldn't be aware of plasma object at all